guides/release/components/template-lifecycle-dom-and-modifiers.md

```diff
@@ -253,35 +253,30 @@ For example, let's say we want to focus an `<input>` in a form as soon as the fo
 inputElement.focus();
 ```
 
-This code needs to run after the element is rendered.
-
-The simplest way to accomplish this is by using the `did-insert` modifier from [@ember/render-modifiers][render-modifiers].
-
-[render-modifiers]: https://github.com/emberjs/ember-render-modifiers
+This code needs to run after the element is rendered, and modifiers provided that capability.
 
 ```handlebars {app/components/edit-form.hbs}
 <form>
-  <input {{did-insert this.focus}}>
+  <input {{this.autofocus}}>
 </form>
 ```
 
+Here we define a local modifier right on the component class.
+
 ```js {app/components/edit-form.js}
 import Component from '@glimmer/component';
-import { action } from '@ember/object';
+import { modifier } from 'ember-modifier';
 
 export default class EditFormComponent extends Component {
-  @action
-  focus(element) {
-    element.focus();
-  }
+  autofocus = modifier((element) => element.focus());
 }
 ```
```

Comment:
I think we should swap the order, putting the modifier declaration first. The reason is that sometimes devs new to Ember copy-paste the first half of examples and expect it to work, then get confused when they see a ton of errors.